src/core/proof.ml

```diff
@@ -65,40 +78,92 @@ type t = proof_state
     [builtins] may be used by tactics, and have been declared. *)
 let init : Pos.strloc -> term -> t = fun name a ->
   let proof_term = fresh_meta ~name:name.elt a 0 in
-  let proof_goals = [Goal.of_meta proof_term] in
-  {proof_name = name; proof_term; proof_goals}
+  {proof_name = name; proof_term; proof_goals = []}
+
+(** [init builtins name a] returns an initial proof state for a theorem  named
```

Comment:
The arguments in the comment need to be fixed.

Reply:
has been fixed since